services/api-dotnet/src/app/Common/Common.csproj

```diff
@@ -8,12 +8,14 @@
 
   <ItemGroup>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
+    <PackageReference Include="AWSSDK.S3" Version="3.7.9.33" />
     <PackageReference Include="BCrypt.Net-Core" Version="1.6.0" />
     <PackageReference Include="FluentValidation" Version="10.4.0" />
     <PackageReference Include="Hangfire.Core" Version="1.7.28" />
     <PackageReference Include="Hangfire.Mongo" Version="1.7.0" />
     <PackageReference Include="Hangfire.PostgreSql" Version="1.9.7" />
     <PackageReference Include="MailKit" Version="3.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="5.0.17" />
```

Comment:
Where it's used?

Reply:
The package contains IFormFile interface, which is used in [UploadPhotoModel.cs](https://github.com/paralect/ship/blob/fdd954323d89439dc78e1d009591f2c30f8b339e/services/api-dotnet/src/app/Common/Models/View/User/UploadPhotoModel.cs)